### ISSUE TITLE
[Bug] File system view without content

### ISSUE THREAD

**Issue by GitMensch** [OpenCobolIDE/OpenCobolIDE#237]
There's no content with current 4.7 preview 2 with some files.

## System information

- OpenCobolIDE: 4.7.dev37
- GnuCOBOL: GNU Cobol 1.1.0
- Python: 3.4.3 (32bits)
- Qt: 5.5.0
- PyQt: 5.5
- pyqode.core: 2.7.0.dev33
- pyqode.cobol: 2.7.dev0
- pyqode.qt: 2.7.dev0
- pygments: 2.0.2
- QDarkStyle: 2.1

## Log:

~~~~~
DEBUG::pyqode.core.managers.file::file open: c:\test.cob
DEBUG::pyqode.core.widgets.splittable_tab_widget::current tab changed (old=CobolCodeEdit(path='\\\\SERVER\\share\\com.cob'), new=CobolCodeEdit(path='C:\\TEST.COB'))
DEBUG::open_cobol_ide.compilers::detecting file type: C:\TEST.COB - cp1252
DEBUG::open_cobol_ide.compilers::file type: <FileType.MODULE: 1>
DEBUG::open_cobol_ide.compilers::detecting file type: C:\TEST.COB - cp1252
DEBUG::open_cobol_ide.compilers::file type: <FileType.MODULE: 1>
INFO::open_cobol_ide.controllers.edit::current editor changed: C:\TEST.COB
DEBUG::pyqode.core.api.folding::(l96) inconsistent fold level, difference between consecutive blocks cannot be greater than 1 (2).
DEBUG::pyqode.core.api.folding::(l111) inconsistent fold level, difference between consecutive blocks cannot be greater than 1 (2).
DEBUG::pyqode.core.api.syntax_highlighter::rehighlight duration: 0.148000s
DEBUG::open_cobol_ide.view.editors.cobol::linter enabled: True
DEBUG::open_cobol_ide.compilers::detecting file type: C:\TEST.COB - cp1252
DEBUG::open_cobol_ide.compilers::file type: <FileType.MODULE: 1>
DEBUG::open_cobol_ide.controllers.view::showing page <Page.EDIT: 1>
~~~~~

**Comment by GitMensch** [comment]
Just to add this bit of information: for some files the file system pane has content, I'm not sure what the reason for this is.

**Comment by ColinDuquesnoy** [comment]
It happens only with UNC paths isn't it? 

QFileSystemModel does not support unc paths. Fixing this is a huge task as it requires to rewrite the entire file system model. I never work with UNC path and I won't fix this.

**Comment by GitMensch** [comment]
Therefore the "fix" is to add the following:

* a warning message when opening from UNC path and
* if possible output of "UNC-Paths not supported" in file system view

Please reopen the issue for this.

And... No, happens with C:\TEST.COB, too.

**Comment by ColinDuquesnoy** [comment]
> No, happens with C:\TEST.COB, too.

Very strange...never occured for me but I'll rechek.

**Comment by ColinDuquesnoy** [comment]
> No, happens with C:\TEST.COB, too.

This happen because the file you open is at the root of your drive. 

The treeview has been designed to show the parent directory as the root node. To accomplish that we actually set the parent directory of the directory that contains the file you opened and ignore all directories except the directory that contains the file.

Solved by showing all files/directories as top level items if the root path is a mount point.

**Comment by ColinDuquesnoy** [comment]
I still have to handle the case of UNC paths.

**Comment by ColinDuquesnoy** [comment]
I added both a warning message box (shown only once per session) and a warning item in the treeview.
Here is how it looks:

![warningbox](https://cloud.githubusercontent.com/assets/1681217/10826928/5f49c6f0-7e6c-11e5-9326-8fe048dd7489.png)

![treeview](https://cloud.githubusercontent.com/assets/1681217/10826937/668b08ca-7e6c-11e5-92f0-47ab3409e7bf.PNG)

**Comment by GitMensch** [comment]
How to say someone "it's not good what you want to do...": do it the Colin way :+1:

**Comment by GitMensch** [comment]
Just a question after seeing the message the first time on this machine.
Shouldn't it be "You are about to open a**n** file using a UNC pathname." (the "n" is missing in the text but I may be wrong about this typo as a non-native speaker).

**Comment by ColinDuquesnoy** [comment]
I am not a native speaker but I think the "n" is not missing because "U" sounds like a "Y" which is a consonant sound.
